### ISSUE TITLE
Units/explanation of info data structure

### ISSUE THREAD

**Issue by debadeepta** [D-X-Y/NATS-Bench#1]
![info](https://user-images.githubusercontent.com/9354770/102655704-9c7e8780-4127-11eb-9481-b32c397e2576.PNG)

Hi, First of all thanks for this tremendous service to the NAS research community. I have a few clarification questions:
What do the following fields in `info = api.get_more_info(arch_index, 'cifar10')` datastructure mean:
`train-per-time`, `train-all-time`, `test-per-time`, `test-all-time`?

![cost_info](https://user-images.githubusercontent.com/9354770/102656235-6392e280-4128-11eb-981e-71fbec3f8377.PNG)

Also in `cost_info = api.get_cost_info(arch_index, 'cifar10')` what do the following field mean?
`T-train@epoch`, `T-train@total`, `T-ori-test@epoch`, `T-ori-test@total`

If these could be documented more explicitly it will be very useful. Thanks again!

**Comment by D-X-Y** [comment]
Thanks for your interest!

First of all, the data split in NATS-Bench is different from that in the official CIFAR paper. In NATS-Bench, we split the original CIFAR-10 training set into two parts, i.e., a training set and a validation set. In the following, we will use the splits of NATS-Bench to explain.
```
# The performance of 0-th architecture on CIFAR-10 (trained by 12 epochs)
info = api.get_more_info(0, 'cifar10', hp=12)
# info['train-loss'] is the loss on the training set of CIFAR-10.
# info['train-all-time'] is the total training time for 12 epochs on CIFAR-10.
# info['train-per-time'] is the per-epoch training time on CIFAR-10. (equals to info['train-all-time'] / 12)
# info['test-all-time'] is the total evaluation time on the test set of CIFAR-10 for 12 times.
# info['test-per-time'] is the evaluation time on the test set of CIFAR-10: {:}'.format())
cost_info = api.get_cost_info(0, 'cifar10')
# cost_info['T-train@epoch']  is the per epoch training cost for CIFAR-10.
# Note that the training set of CIFAR-10 in NATS-Bench is a subset of the original training set in CIFAR paper.
# cost_info['T-train@total'] = cost_info['T-train@epoch'] * 12
# cost_info['T-ori-test@epoch'] is the time cost for the evaluation on the test set of CIFAR-10.
```
Please let me know if you have any other question.
